server/app/auth/oidc/OidcProvider.java

```diff
@@ -197,17 +199,19 @@ public OidcClient get() {
     if (providerName.isPresent()) {
       client.setName(providerName.get());
     }
-
     client.setCallbackUrl(callbackURL);
     client.setProfileCreator(getProfileAdapter(config, client));
     client.setCallbackUrlResolver(new PathParameterCallbackUrlResolver());
+    client.setLogoutActionBuilder(
+        new CiviformOidcLogoutActionBuilder(civiformConfig, config, clientID));
+
     try {
       client.init();
     } catch (Exception e) {
       logger.error("Error while initilizing OIDC provider", e);
       throw e;
     }
-
+    logger.debug("Client: " + client.toString());
```

Comment:
Same question / concern as above. Should we be logging this information and are confident that it won't contain any sensitive data?